.github/workflows/client.yml

```diff
@@ -33,3 +33,12 @@ jobs:
 
       - name: Build
         run: ./gradlew compileDebugAndroidTestSources check assemble ktlintCheck
+
+      - name: Upload APKs
+        uses: actions/upload-artifact@v3
+        with:
+          name: seedvault-${{ github.sha }}-apks
+          path: |
+            app/build/outputs/apk/debug/app-debug.apk
+            contactsbackup/build/outputs/apk/debug/contactsbackup-debug.apk
+            storage/demo/build/outputs/apk/debug/demo-debug.apk
```

Comment:
Is there a way to give those APKs a nicer name, maybe even include commit hash or something like that?

Reply:
A there's a folder name already. Maybe that's good enough.

Reply:
For options see https://github.com/actions/upload-artifact

Could also have individual artifacts but it's still be apk inside zip, so I figured might as well combine.

I'm ok with anything really.